### ISSUE TITLE
release plan

### ISSUE THREAD

**Issue by stefanpenner** [ember-cli/ember-cli#5028]
I've been focused on other things, but it's long overdue for a release. Although others can release, I have likely been blocking because I have not provided sufficient guidance.

My thoughts are as follows, and would appreciate a hand if anyone has spare cycles.

Release v1.13.9 (end of this week)

- [x] branch 1.13.x – @twokul 
- [x] latest 1.13.x deps – @twokul 
- [x] change log (with tests/index.html change) – @twokul 
- [x] bump broccoli-funnel / merge-trees to ^1.0.0 - will likely require them released as well.
- [x] fix up/work around npm v3 defeating our bundling of non native dependencies
- [x] native dependencies fix needs to transition from `git` deps to release `npm` packages -> https://github.com/airportyh/testem/issues/660
- [x] ??

Release v2.x (end of next week)

- [x] figure out which version it should be
- [ ] addonified ember/ember-data
- [ ] no more core bower dependencies
- [ ]  ???

**Comment by chadhietala** [comment]
:ship:

**Comment by bartocc** [comment]
A new release is definitely needed. Not seeing ember-cli releases lately felt strange while ember was moving.

Will this release include #5015 ?

**Comment by stefanpenner** [comment]
@bartocc unlikely.

**Comment by rtablada** [comment]
@stefanpenner will Node v4 warning be cut in these releases?

**Comment by rtablada** [comment]
Also what's "no more core bower dependencies" is Ember CLI moving to recommending NPM deps for browser requirements?

**Comment by stefanpenner** [comment]
> @stefanpenner will Node v4 warning be cut in these releases?

yup

**Comment by stefanpenner** [comment]
> Also what's "no more core bower dependencies" is Ember CLI moving to recommending NPM deps for browser requirements?

yes

**Comment by patocallaghan** [comment]
@stefanpenner I'd be happy to give a dig out with the release

It would also be good to try get the Watchman v4 PR merged (#5026) as that's the version installed by default with `brew install watchman`

**Comment by stefanpenner** [comment]
> It would also be good to try get the Watchman v4 PR merged (#5026) as that's the version installed by default with brew install watchman

@patocallaghan agreed, once @rwjblue's comments have been addressed we can move forward with it.

**Comment by twokul** [comment]
I can take a look at upgrading to latest 1.13.x deps

**Comment by stefanpenner** [comment]
> I can take a look at upgrading to latest 1.13.x deps

branching to 1.13.x & updating deps?

**Comment by twokul** [comment]
@stefanpenner ya

**Comment by stefanpenner** [comment]
> fix up/work around npm v3 defeating our bundling of non native dependencies

@patocallaghan if you have cycles looking into ^^ would be of great help.

TL;DR pre npm 3.0, I manually patched out optional native dependencies via https://github.com/ember-cli/ember-cli/blob/master/bin/prepare-release#L8-L9 in v3 this hack has been defeated.

It also appears in v3 we are pwn'd by a bug (that may now be fixed) that causes the install to fail if a optional dependency fails. (this needs further investigation)

Ultimately the goal of ^, is to remove the native dependencies. I would be very happy to consider any reasonable proposal that accomplishes this. Including pure forks of the offending libraries.

**Comment by patocallaghan** [comment]
@stefanpenner sure, I can look at that. Not *entirely* sure what'll be involved but will have a go anyway.

**Comment by twokul** [comment]
I created a new [1.13.x branch](https://github.com/ember-cli/ember-cli/tree/1.13.x)

**Comment by nbibler** [comment]
@twokul: What's the intended difference between the new [1.13.x](https://github.com/ember-cli/ember-cli/tree/1.13.x) branch and the existing [stable-1-13](https://github.com/ember-cli/ember-cli/tree/stable-1-13) branch?

**Comment by twokul** [comment]
@nbibler @stefanpenner updated [stable-1-13](https://github.com/ember-cli/ember-cli/tree/stable-1-13) branch.

**Comment by devinus** [comment]
Which versions of Ember and Ember Data will `1.13.9` target? What about `2.x`?

**Comment by btecu** [comment]
@devinus `ember 1.13.10` and `ember data 1.13.14`.
https://github.com/ember-cli/ember-cli/commit/f04c4662adfc316809505e2647e12e52462e0d07

**Comment by wldcordeiro** [comment]
Might be RFC worthy (or already existing) but moving from jshint to eslint in the 2.x series sounds like a good idea.

**Comment by stefanpenner** [comment]
Jshint to eslint by default will likely not happen till 3.0 unless something makes a seamless upgrade story

**Comment by btecu** [comment]
Isn't eslint slower than jshint?

**Comment by ReyAyanami** [comment]
What about Release v2.x?

**Comment by stefanpenner** [comment]
@ReyAyanami the checklist is accurate

**Comment by masak** [comment]
Is release 2.x likely to show up soon? Is there something in particular that's blocking it?

**Comment by stefanpenner** [comment]
@masak the above checklist
